.gitmodules

```diff
@@ -6,10 +6,6 @@
 	path = deps/eigen
 	url = https://gitlab.com/libeigen/eigen.git
 	shallow = true
-[submodule "deps/backward-cpp"]
```

Comment:
I don't like it because
- stack traces in general don't provide enough information for debugging. You need to look around at variables with gdb to know the root cause of the bug, and if you do that then the backtrace is useless because gdb does it better
- users don't care, only developers do, and developers should rather use gdb
- when enabled, it pollutes the output because it dumps the whole stack trace for a simple exception, so a simple error message is

Reply:
To be continued :D